content/docs/intro/languages/javascript.md

```diff
@@ -78,9 +78,9 @@ When using Pulumi's built in TypeScript support, a `tsconfig.json` file is optio
 {
     "compilerOptions": {
         "outDir": "bin",
-        "target": "es6",
+        "target": "es2016",
         "lib": [
-            "es6"
+            "es2016"
```

Comment:
This whole `lib` setting could be deleted.

Reply:
Or just replace this whole snippet with the `tsconfig.json` from the TypeScript templates: https://github.com/pulumi/templates/blob/master/typescript/tsconfig.json